numpy/linalg/tests/test_linalg.py

```diff
@@ -682,6 +682,12 @@ def test_empty_identity(self):
         assert_equal(vh.shape, (4, 4))
         assert_equal(vh, np.eye(4))
 
+    def test_svdvals(self):
```

Comment:
`numpy/linalg/tests/test_regression.py` has a `test_svd_no_uv`, maybe add some logic to that test that ensures the `w` returned by linalg.svd` in that test is the same as what `linalg.svdvals` returns?

Reply:
Sure! I added `svd`/`svdvals` check in `test_svd_no_uv`.